game.js

```diff
@@ -1,12 +1,20 @@
-const gameSpace = document.getElementsByClassName('main__container')[0]
+const gameSpace = document.getElementsByClassName('main__container')
+const [container] = gameSpace
 
-const gameCards = Array.from(Array(20))
-gameCards.forEach(() => {
+Array
+.from(Array(20))
```

Comment:
tab, you can't put `.` directly below `Array`. Tab means that you want to chain it